README.md

```diff
@@ -5,6 +5,13 @@ Simcard management for GLPI
 * Updated by Anthony Piesset and Dethegeek for GLPi 0.84
 * Updated by Thierry Bugier Pineau for GLPi 0.85 and 0.90
 
+Integration in GLPI 9.4
+-----------------------
+
+Beside what they wrote, the way that GLPI handle SIM Card in core modules is really awful.
```

Comment:
Just a note: I do not remember you opened any pull request on GLPI in order to get that fixed. Not sure you even open an issue explainning what could be improved.

Also note there have been development for the 9.5 on that point. Maybe you should this time early test those changes and contribute to get it better if needed?

Reply:
My wrong. I pulled for merge but I didn't want this.
My point is that I need the way SIM plugin work, threating the items like a computer or device.
Actually, the SIM mode inside GLPI core (imho) need to be completly rewritten, cause, for example, the search mode (using global search for a number doesn't work).

So, if you want, u can delete my (wrong) pull request and we can work together for change the way GLPI threat SIM Card